02_bubble/nicholeL/main_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestBubbleSort(t *testing.T) {
+	testCases := map[string]struct {
+		input    []int
+		excepted []int
+	}{
+		"one":   {[]int{3, 2, 1, 5}, []int{1, 2, 3, 5}},
```

Comment:
include single element slice, slice with more identical elements, already sorted and reverted slice.

Reply:
will add more test case